contracts/Market.sol

```diff
@@ -681,12 +687,10 @@ contract Market is
     bytes32 s
   ) external whenNotPaused {
     _validateCertificateAmount({amount: amount});
-    (
-      uint256 countOfRemovalsAllocated,
-      uint256[] memory ids,
-      uint256[] memory amounts,
-      address[] memory suppliers
-    ) = _allocateRemovals({purchaser: _msgSender(), certificateAmount: amount});
+    SupplyAllocationData memory allocationData = _allocateRemovals({
+      purchaser: _msgSender(),
```

Comment:
Does this param still make sense? Looks like it's always set to msgSender now.

Reply:
good catch